dotnet/example_code/ec2/Ec2SpotCRUD.cs

```diff
@@ -0,0 +1,220 @@
+//snippet-sourcedescription:[Ec2SpotCRUD demonstrates how to create, cancel, and terminate a spot instance request using Amazon EC2.]
+//snippet-keyword:[dotnet]
+//snippet-keyword:[.NET]
+//snippet-sourcesyntax:[.net]
+//snippet-keyword:[Code Sample]
+//snippet-keyword:[Amazon Elastic Compute Cloud]
+//snippet-keyword:[Amazon EC2]
+//snippet-service:[ec2]
+//snippet-sourcetype:[full-example]
+//snippet-sourcedate:[Sep 20, 2019]
+//snippet-sourceauthor:[Doug-AWS]
+/*******************************************************************************
+* Copyright 2009-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"). You may
+* not use this file except in compliance with the License. A copy of the
+* License is located at
+*
+* http://aws.amazon.com/apache2.0/
+*
+* or in the "license" file accompanying this file. This file is
+* distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the specific
+* language governing permissions and limitations under the License.
+*******************************************************************************/
+/* Build 
+   Don't forget to use the Visual Studio command prompt
+   Tested with csc /version == 2.10.0.0
+     (C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\15.0\bin\Roslyn)
+   AWSSDK.Core.dll
+     (C:\Users\USERNAME\.nuget\packages\awssdk.core\3.3.100\lib\net45)
+   AWSSDK.EC2.dll
+     (C:\Users\USERNAME\.nuget\packages\awssdk.ec2\3.3.130.2\lib\net45)
+
+   csc Ec2SpotCRUD.cs -reference:AWSSDK.Core.dll -reference:AWSSDK.EC2.dll
+
+   Get the list of AWSSDK Nuget packages at:
+       https://www.nuget.org/packages?q=AWSSDK&prerel=false
+ */
+// snippet-start:[ec2.dotnet.spot_instance_using]
+using System;
+using System.Collections.Generic;
+using System.Threading;
+
+using Amazon.EC2;
+using Amazon.EC2.Model;
+// snippet-end:[ec2.dotnet.spot_instance_using]
+
+namespace Ec2SpotCrud
+{
+    class Program
+    {
+        // snippet-start:[ec2.dotnet.spot_instance_request_spot_instance]    
+        public static SpotInstanceRequest RequestSpotInstance(
```

Comment:
Error handling in this--and all other sibling--methods?

Reply:
The only error handling I do is some sanity checks on the inputs.